src/ddmd/expression.d

```diff
@@ -8413,6 +8413,22 @@ extern (C++) final class ImportExp : UnaExp
             ob.writestring(")");
             ob.writenl();
         }
+        if (global.params.makeDeps !is null)
```

Comment:
I've added an import expression to the test file, this should cover these lines.